README.md

```diff
@@ -74,7 +74,7 @@ Yes, we use GS Collections in many of our internal applications.
 ## Acquiring GS Collections
 
 ### Download
-* [Version 6.2.0](http://www.goldmansachs.com/gs-collections/binaries/gs-collections-6.2.0.zip)
+* [Version 7.0.0](https://github.com/goldmansachs/gs-collections/releases/tag/7.0.0)

```

Comment:
At time of writing, `http://www.goldmansachs.com/gs-collections/binaries/gs-collections-7.0.0.zip` gives me a 404, so I changed it to the 7.0.0 tag in the releases page.